src/Logary/Internals.fs

```diff
@@ -205,6 +221,43 @@ module internal Set =
     | (s : _ Set) when s.Count = 0 -> Some EmptySet
     | _ -> None
 
+type Timeout =
```

Comment:
We probably don't need this anymore because it's built into Hopac in a nice way. Don't you think?